lib/Bio/EnsEMBL/DataCheck/Checks/MetaKeyConditional.pm

```diff
@@ -185,6 +185,8 @@ sub repeat_analysis {
         species_id = $species_id
       AND
         logic_name <> "repeatmask_repeatmodeler"
+      AND
```

Comment:
Personaly, I would have created a list, and use the list in the query instead of coupled "AND". Personal point of view.

Reply:
I agree, but this is clear enough, for now at least. If we get more exceptions in the future, a list would be neater.